ci_framework/tests/integration/targets/make/files/get_makefiles_env/expected_variables_values.yml

```diff
@@ -46,7 +46,7 @@ variables:
   CINDER_CR: "/home/test-user/out/operator/cinder-operator/config/samples/cinder_v1beta1_cinder.yaml"
   CINDER_IMG: "quay.io/openstack-k8s-operators/cinder-operator-index:latest"
   CINDER_KUTTL_CONF: "/home/test-user/out/operator/cinder-operator/kuttl-test.yaml"
-  CINDER_KUTTL_DIR: "/home/test-user/out/operator/cinder-operator/tests/kuttl/tests"
+  CINDER_KUTTL_DIR: "/home/test-user/out/operator/cinder-operator/tests/kuttl/test"
```

Comment:
looking at the other review [1] you still have tests/kuttl/tests directory eg tests/kuttl/tests/cinder_scale/05-cleanup-cinder.yaml 

so is this bit right or do we want to keep tests/ or do you want to change the cinder operator patch?

[1] https://github.com/openstack-k8s-operators/cinder-operator/pull/264/commits/b49b218e4c9d1804d75000ce2cbccc9ed5dacae4#diff-770a09a99b77b69cd9845bbc37e8516bf5755acc16d78eacf84e84d667b01ca0

Reply:
I am removing this change - this seemed to have no effect on the cinder_kuttl tests. Will focus on cinder in a separate PR